german/the-black-hack.md

```diff
@@ -259,8 +260,8 @@ Traditionelle OSR-Spiele bilden magische Gegenstände ab, indem sie +1 oder +2 a
 
 **Start-TP :** W10 + 4
 **TP pro Stufe/Erholung :** 1W10
-**Waffen & Rüstung :** Alle Waffen und Rüstungen
-**Angriffsschaden :** 1W8 / 1W6 Unbewaffnet oder Improvisiert
+**Waffen & Rüstung :** alle Waffen und Rüstungen
+**Angriffsschaden :** 1W8 / 1W6 unbewaffnet oder improvisiert
```

Comment:
Hmm, ich kenne die anderen Sprachen zu wenig, um zu sehen, wie es da gemacht wurde. Aber womöglich gibt es da ebenfalks keine gemeinsame Linie. Ich würde regelrelevante Begriffe in jedem Fall hervorheben wollen...